### PR TITLE
Add close button to sidebar

### PR DESCRIPTION
**Pull request by georapbox**
Fixes issue  #37

### CODE REVIEW

File: src/i18n/de-DE.js [hunk 1]
```diff
@@ -70,6 +70,7 @@ const translations = {
   considerDPR: 'Pixelverhältnis des Geräts beachten',
 
   tweak: 'Anpassen',
+  closeOptions: 'Schließung',
```
Comment (tomayac) [inline src/i18n/de-DE.js:73]:
```suggestion
  closeOptions: 'Schließen',
```

But very close 😃

**Review by tomayac — COMMENTED** [review]
I’ll test this tomorrow. Thanks so much for working on this! `<3`

File: src/i18n/de-DE.js (hunk 1 above)
Comment (georapbox) [reply src/i18n/de-DE.js]: I trusted in Google translate 😂

File: index.html [hunk 2]
```diff
@@ -141,6 +141,7 @@
       <details class="main">
         <summary></summary>
         <div class="sidebar">
+          <button type="button" class="close-options-button">&#10005;</button>
```
Comment (tomayac) [inline index.html:144]:
```suggestion
          <button type="button" class="close-options-button">
            <span aria-hidden="true">&#10005;</span>
          </button>
```

File: src/js/ui.js [hunk 3]
```diff
@@ -417,6 +418,7 @@ const changeLanguage = () => {
   summary.innerHTML = '';
   summary.append(createIcon(optionsIcon));
   summary.append(document.createTextNode(i18n.t('tweak')));
+  closeOptionsButton.title = i18n.t('closeOptions');
```
Comment (tomayac) [inline src/js/ui.js:421]:
```suggestion
  closeOptionsButton.ariaLabel = i18n.t('closeOptions');
```

**Review by tomayac — APPROVED** [review]
LGTM

Thank you!

**Pull request merged**